eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -350,6 +359,34 @@ object Helpers {
         data.commitments.remoteCommit.spec.toRemoteMsat == 0 &&
         data.commitments.remoteNextCommitInfo.isRight
 
+    /**
+      * Check if a channel is closed (i.e. its closing tx has been confirmed)
+      * @param data channel state data
+      * @param closingTx closing transaction, needed in case of mutual close
```

Comment:
This tx isn't necessarily the closing tx, so the attribute name and description is misleading. It can be any confirmed tx, and if it happens to be a closing tx then we will consider this a `MutualClose`